src/iuse.cpp

```diff
@@ -3355,9 +3355,9 @@ int iuse::pickaxe(player *p, item *it, bool, const tripoint& )
     int turns;
     if (g->m.is_bashable(dirx, diry) && (g->m.has_flag("SUPPORTS_ROOF", dirx, diry) || g->m.has_flag("MINEABLE", dirx, diry)) &&
         g->m.ter(dirx, diry) != t_tree) {
-        // Takes about 100 minutes (not quite two hours) base time.  Construction skill can speed this: 3 min off per level.
+        // Takes about 90 minutes (an hour and a half) base time.  Construction skill can speed this: 4 min off per level.
         ///\EFFECT_CARPENTRY speeds up mining with a pickaxe
-        turns = (100000 - 3000 * p->get_skill_level( skill_carpentry ));
```

Comment:
Hmm. That produces a massive time nerf for unskilled use of a pickaxe, while making the benefit of construction skill much higher. Seems acceptable though, given players are unlikely to begin mining in earnest unless they already have the 5 skill needed to dig their own mine where they wish. This is also far much saner than what I was testing:

Level | Minutes
------------ | -------------
0 | 90
1 | 81.818
2 | 75
3 | 69.23
4 | 64.285
5 | 60
6 | 56.25
7 | 52.941
8 | 50
9 | 47.368
10 | 45